src/components/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  useEffect,
+  useState,
+  memo,
+  useMemo,
+} from 'react';
+import { useParams, useSearchParams } from 'react-router-dom';
+
+import { PeopleFilters } from '../PeopleFilters';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { getFilteredPeople } from '../../utils/getFilteredPeople';
+
+export const PeoplePage: React.FC = memo(() => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [hasError, setHasError] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isLoaded, setIsLoaded] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState<boolean>(false);
  const [isLoaded, setIsLoaded] = useState<boolean>(false);
```
I think here are similiar things, !isLoading === isLoaded, maybe better use one of this